### PR TITLE
Remove facilitator templates, mailers, and controllers

### PR DESCRIPTION
**Pull request by megcrenshaw**
Removes facilitator templates, mailers, and controllers, and their associated test / preview files.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/app/models/pd/fit_weekend1819_registration.rb [hunk 1]
```diff
@@ -25,16 +25,11 @@ class Pd::FitWeekend1819Registration < ApplicationRecord
   belongs_to :pd_application, class_name: 'Pd::Application::ApplicationBase', optional: true
 
   after_create :update_application_status
-  after_create :send_fit_weekend_confirmation_email
 
   def update_application_status
```
Comment (TurnerRiley) [inline dashboard/app/models/pd/fit_weekend1819_registration.rb:31]: If the controller for the weekend registration is being deleted, what is the need for keeping the rest of these files?

**Review by TurnerRiley — APPROVED** [review]
Yes!!!!!! Look at all that red! Just had 1 quick question!

File: dashboard/app/models/pd/fit_weekend1819_registration.rb (hunk 1 above)
Comment (TurnerRiley) [reply dashboard/app/models/pd/fit_weekend1819_registration.rb:31]: Sorry I probably should've specified, the "these files" I'm referring to are the files you modified but didn't delete

Comment (megcrenshaw) [reply dashboard/app/models/pd/fit_weekend1819_registration.rb:31]: I posted here https://codedotorg.slack.com/archives/C0T10HG6N/p1670258684933639 about deleting the model files. I'm not sure it's kosher to delete the model files when we still want to keep the table around –– so for now, I'm keeping the models until I learn what the best practice is

Comment (TurnerRiley) [reply dashboard/app/models/pd/fit_weekend1819_registration.rb:31]: Sounds good! Would it be possible to add the comment in those files that Ben suggested just for future reference on why the files are "gutted"? This is just a comment though so maybe not worth a full rebuild, totally your call!

Comment (megcrenshaw) [reply dashboard/app/models/pd/fit_weekend1819_registration.rb:31]: Good call! I'm going to see if it's possible to remove all of the model files. If it is, II won't add anything here. If it's not, I'll remove everything from the model files other than the annotations with a comment explaining the dealio

Comment (megcrenshaw) [reply dashboard/app/models/pd/fit_weekend1819_registration.rb:31]: I went ahead and adding todo comments in the models!

**Pull request merged**